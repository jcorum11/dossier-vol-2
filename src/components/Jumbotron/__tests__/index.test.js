import React from "react"
import { render, cleanup } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"
import Hero from ".."

afterEach(cleanup)

it("renders", async () => {
  render(<Hero />)
})

it("matches snapshot", async () => {
  const { asFragment } = render(<Hero />)
  expect(asFragment()).toMatchSnapshot()
})
