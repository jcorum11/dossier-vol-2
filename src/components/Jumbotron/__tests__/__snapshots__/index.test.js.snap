// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-gsDKAQ euceyN"
  >
    <div
      class="sc-bdvvtL fmtvOV"
    >
      <div
        class="sc-dkPtRN GvWfk"
      />
    </div>
  </div>
</DocumentFragment>
`;
